vendor/vendor.json

```diff
@@ -790,6 +790,12 @@
 			"version": "v1.15.87",
 			"versionExact": "v1.15.87"
 		},
+		{
```

Comment:
FYI dependencies are managed by `govendor` (for a little while longer). The dependencies required by this new service will be pulled in via #6661 and the change to the vendor.json should be removed from this PR. 👍

Reply:
Cool 👍